x-pack/plugins/security_solution/server/lib/timeline/routes/timelines/import_timelines/index.test.ts

```diff
@@ -257,25 +257,13 @@ describe('import timelines', () => {
     test('should provide no noteSavedObjectId when Creating notes for a timeline', async () => {
       const mockRequest = await getImportTimelinesRequest();
       await server.inject(mockRequest, context);
-      expect(mockPersistNote.mock.calls[0][1]).toBeNull();
-    });
-
-    test('should provide new timeline version when Creating notes for a timeline', async () => {
```

Comment:
@XavierM it didn't look like the version was being used so we removed it. There isn't something that we missed right?

Reply:
I chatted with Xavier about this it we think it's fine to remove the version since we weren't using it.